sagemaker-training-compiler/huggingface/pytorch_single_gpu_single_node/roberta-base/roberta-base.ipynb

```diff
@@ -58,7 +58,7 @@
    "source": [
```

Comment:
<p>Same question:</p><p> </p><p>Do we need to install torch 1.12 instead of torch 1.13.1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3812/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Yes. You are right, we don't need to install torch here. I will remove this for all of three notebooks.